x/gov/keeper/proposal.go

```diff
@@ -63,6 +66,9 @@ func (keeper Keeper) SubmitProposal(ctx context.Context, messages []sdk.Msg, met
 			return v1.Proposal{}, errorsmod.Wrapf(types.ErrInvalidSigner, sdk.AccAddress(signers[0]).String())
 		}
 
+		// TODO HV2: all other valid sdk.Msg will end up here
```

Comment:
Wdym you won't have a proper handler for them, there's no change to the code where proposal messages get executed https://github.com/0xPolygon/cosmos-sdk/blob/mardizzone/POS-2129-gov/x/gov/abci.go#L191-L200

Reply:
Same as before, let's discuss in the call, thanks

Reply:
This is outdated, will close